pandas/_typing.py

```diff
@@ -25,7 +25,7 @@
     Optional,
     Sequence,
     Tuple,
-    Type,
+    Type as Type_T,
```

Comment:
why not TypeT

Reply:
or type_t ?

Reply:
you're right, `type_t` would be more consistent

Reply:
So I will change the variable name to type_t, keep the changes to 15-20 files as of now, and still keep the --keep-runtime-typing flag there in precommit hook.
Is that right @MarcoGorelli ?

Reply:
yes, that's right @fathomer! Then, in the final PR, you can remove the flag

Reply:
I have reverted changes to all other files except some 20 files for this PR, did a rebase to upstream/master and pushed.
For the remaining files do you want me raise the other PRs just now or wait until this one gets merged ?

Reply:
Thanks @fathomer! You can raise the others now if you want, I'll look over them tomorrow

Reply:
@MarcoGorelli @jreback Raised the other PRs as well, for some reason not getting the option to add you guys as reviewers there. Please feel free to review them.